include/swift/Reflection/TypeRef.h

```diff
@@ -283,6 +283,24 @@ class NominalTypeRef final : public TypeRef, public NominalTypeTrait {
 class BoundGenericTypeRef final : public TypeRef, public NominalTypeTrait {
   std::vector<const TypeRef *> GenericParams;
 
+  static uint64_t
```

Comment:
Doxygen comment?

Reply:
(What's not obvious from the name is that this counts the generic parameters in the parent only.)